javascript-bank/account.js

```diff
@@ -0,0 +1,46 @@
+/* exported Account */
+
+
+
+function Account(number, holder){
+    this.number = number;
+    this.holder = holder;
+    this.transactions = [];
+}
+
+Account.prototype.deposit = function(amount){
+    // console.log("deposit amount: ", amount);
+    // console.log("deposit amount is integer:", Number.isInteger(amount))
+    if (amount > 0 && Number.isInteger(amount)){
+        this.transactions.push(new Transaction('deposit', amount))
+        return true;
+    } else {
+     return false;
+    }
+}
+
+Account.prototype.withdraw = function(amount){
+    // console.log("withdraw amount: ", amount);
+    // console.log("withdraw amount is integer:", Number.isInteger(amount))
+    if (amount > 0 && Number.isInteger(amount)){
+        this.transactions.push(new Transaction('withdrawal', amount))
+        return true;
+    } else {
+        return false;
+    }
+}
+
+Account.prototype.getBalance = function(){
+    var balance = 0;
+    if (this.transactions.length === 0){
+        return 0;
+    }
```

Comment:
This conditional shouldn't be necessary. If the length is `0`, your loop would never start, so you would still end up returning `0` after the loop attempts to run